CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [15.5.3] - 2025-XX-XX
+* Fixed backup errors with USB when file and app backup are both on
+* Fixed overdue backups not automatically starting when USB drives are plugged in
+* Fixed "System apps" not being backed up
```

Comment:
I think the fix here was just visual: show size of each app backup on restore screen.